libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.m

```diff
@@ -197,6 +186,7 @@ - (void)send:(SFRestRequest *)request delegate:(id<SFRestDelegate>)delegate shou
         [[SFAuthenticationManager sharedManager] loginWithCompletion:^(SFOAuthInfo *authInfo, SFUserAccount *userAccount) {
             __strong typeof(weakSelf) strongSelf = weakSelf;
             [strongSelf enqueueRequest:request delegate:delegate shouldRetry:shouldRetry];
+            [SFUserAccountManager sharedInstance].currentUser = userAccount;
```

Comment:
Yes - I support that. Ideally, I'd like for us to make `SFUserAccountManager` the way to do anything auth related - login/logout/switch user.